crates/compiler/build/src/link.rs

```diff
@@ -784,16 +769,6 @@ fn find_in_folder_or_subfolders(path: &PathBuf, folder_to_find: &str) -> Vec<Dir
     matching_dirs
 }
 
-fn get_target_str(target: &Triple) -> &str {
-    if target.operating_system == OperatingSystem::Windows
-        && target.environment == target_lexicon::Environment::Gnu
-    {
-        "x86_64-windows-gnu"
-    } else {
-        "native"
-    }
-}
-
```

Comment:
I thought there was some workaround-related reason we were doing this...do we not need the workaround anymore?

Reply:
I tested it on Windows with just native and I couldn't see any difference. I haven't tested this PR, though I can do that if that would help.